src/window.rs

```diff
@@ -380,6 +395,39 @@ impl Window {
     pub fn set_decorations(&self, decorations: bool) {
         self.window.set_decorations(decorations)
     }
+        /// Centers the window on the current monitor the window lives in.
+    pub fn center(&self) {
+        self.set_centered(self.get_current_monitor())
+    }
+
+    /// Centers the window on the specified target monitor, falls back to primary monitor if out range.
+    pub fn set_center_monitor(&self, monitor_target: usize) {
+        if monitor_target < self.get_available_monitors().count() {
+            self.set_centered(self.get_available_monitors().nth(monitor_target).unwrap())
```

Comment:
This should be accepting a `Option<MonitorId>`, not usize. 
`self.set_centered(monitor_target.unwrap_or_else(|_| self.get_primary_monitor()))`

Reply:
well, the goal with set_center_monitor() is to only supply a simple monitor index, and if that is out of range then it falls back to the primary monitor, set_center_monitor is better renamed to set_target_monitor() and monitor_target > monitor_index to reflect that idea better. otherwise, if you want to use a MonitorID then simply use set_centered() instead.

Reply:
dropping set_center_monitor() as set_centered() already do this.